enhancements/dns/plugins.md

```diff
@@ -97,24 +98,28 @@ resolution, it restricts how cluster operators manage DNS name queries.
 
 ## Proposal
 
-`PluginConfig` defines one or more CoreDNS plugins that can be configured. If a plugin is not defined, it will use a
-default configuration provided by the cluster-dns-operator. A plugin is represented as a slice when multiple instances
-of the plugin can be expressed within the DNS configuration.
+`Server` defines the schema for a server that runs per instance of CoreDNS. A `Server` must have a `Name` that is unique
```

Comment:
Maybe this is just an implementation note: how will this validation be enforced?

Reply:
I included additional detail on the "Implementation Details Section":
```
`Name` must comply with the [rfc6335](https://tools.ietf.org/rfc/rfc6335.txt) Service Name Syntax.
```

Implementation will be similar to how k8s [validates](https://godoc.org/k8s.io/apimachinery/pkg/util/validation#IsValidPortName) the name of a `ContainerPort`.